### PR TITLE
{bio}[GCC/8.2.0-2.31.1] trimAl 1.4.1

### PR DESCRIPTION
**Pull request by omula**

### CODE REVIEW

File: easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['cb8110ca24433f85c33797b930fa10fe833fa677825103d6e7f81dd7551b9b4e']
+
+start_dir = 'source'
+
+files_to_copy = [(['trimal', 'readal', 'statal'], 'bin')]
+
+github_account = 'scapella'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCC-8.2.0-2.31.1.eb]: @omula can you move this up above `source_urls` (makes it clearer what it's for...)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e53dc8c0b6ca58e23183bdc2dff99697 for a full test report.

File: easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCC-8.2.0-2.31.1.eb (hunk 1 above)
Comment (omula) [reply easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCC-8.2.0-2.31.1.eb]: Done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/480e8cb1b19cd67f5d73aec88eae7387 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2616.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0b89f8dfcac0a58c7483ed1acb3e3ea1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2092.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8d84ff32b10f32e2c294216357092bde for a full test report.

**Comment by boegel** [comment]
Going in, thanks @omula!

**Pull request merged**